### ISSUE TITLE
Double results

### ISSUE THREAD

**Issue by BY-GitHub** [hhelibeb/Wox.Plugin.Firefox-Bookmarks#2]
My suggestion to this bug in firefox.py:

Only 1 query instead of 3:

           results.append(f'''select distinct b.title, visit_count, url, frecency, b.id
                                  from moz_bookmarks as b
                                  join moz_places    as p on b.fk = p.id
                                  where lower(b.title) like {cond2} 
                                     or lower(p.url) like {cond2}                                   
                                  order by frecency desc '''

**Comment by hhelibeb** [comment]
Thank you! I actually forgot how to use it and failed to install it 😥. Now, I'm just working with 'Everything' instead of Wox. Please feel free to take any action you'd like, such as creating a PR...

**Comment by BY-GitHub** [comment]
Do you have a bookmark search functionality for Firefox in "Everything"?

**Comment by hhelibeb** [comment]
> Do you have a bookmark search functionality for Firefox in "Everything"?

no, it only supports file search. for browser history/bookmark i'm using address bar of firefox and lazy to change. well, maybe we firefox users are too marginal...
